src/rapier_body_utils_2d.cpp

```diff
@@ -15,7 +14,7 @@ bool RapierBodyUtils2D::body_motion_recover(
 		Rect2 &p_body_aabb) {
```

Comment:
If you want to pass the marign_aabb (which I don't think you can, see the comment below), rename this to p_margin_aabb